samples/openthread/coap_client/src/ble_utils.c

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
Instead of calling this file `ble_utils.c`, it be more clear to just call it `ble.c`?

Reply:
Do you agree that it is discusional? For me `ble_utils` is better name for that. On the other hand I'm not very attached to it and if you still insist on changing it, it's fine for me too. So, do you insist?

Reply:
I'm fine with keeping it as is. My initial thought when I saw this file was thought when seeing this file name was that it contained some additional layer required to run BLE. When looking at the content, it looks more like it contains the BLE application in itself.

Reply:
Your first impression was good. It is a separated part of the sample which can be reused in the other if needed. The sample real logic is included in callbacks. I'm glad that you don't insist, thank you for understanding.